driver-core/src/main/com/mongodb/internal/connection/ClientMetadataHelper.java

```diff
@@ -168,8 +180,7 @@ static boolean clientMetadataDocumentTooLarge(final BsonDocument document) {
         new BsonDocumentCodec().encode(new BsonBinaryWriter(buffer), document, EncoderContext.builder().build());
         return buffer.getPosition() > MAXIMUM_CLIENT_METADATA_ENCODED_SIZE;
     }
-
-    private enum Environment {
+    private enum FaasEnvironment {
```

Comment:
With the introduction of new parameters in the 'env' section related to containers/orchestration, I've renamed this enum in accordance with the [specification](https://github.com/abr-egn/specifications/blob/c4831b426c7eab8749723d7f3cd6bb5c237bcbbe/source/mongodb-handshake/handshake.rst#faas) as contains metadata for FaaS.

Reply:
Ack